spec/controllers/projects_controller_spec.rb

```diff
@@ -117,6 +117,12 @@
         expect(response).to have_http_status(:unprocessable_entity)
         expect(response.content_type).to eq 'application/json'
       end
+
+      it 'never calls create activity service' do
```

Comment:
Is this named correctly?

Reply:
Nice catch. This was implemented just for testing, will be fixed in the next branch/PR